flow/include/flow/genericactors.actor.h

```diff
@@ -571,26 +571,26 @@ class AsyncMap : NonCopyable {
 			setUnconditional(k, v, i);
 	}
 	void setUnconditional(K const& k, V const& v) { setUnconditional(k, v, items[k]); }
+
+	void sendError(K const& begin, K const& end, Error const& e) {
+		if (begin >= end)
+			return;
+		std::vector<Promise<Void>> noDestroy = swapRangePromises(items.lower_bound(begin), items.lower_bound(end));
```

Comment:
Nit: the name `noDestroy` here is now a bit confusing, maybe we could switch it back to `ps`